src/main/webapp/css/welcome.css

```diff
@@ -0,0 +1,301 @@
+:root {
+    --text-01: #000000;
+    --light-01: #F9F9F9;
+    --light-02: #FFFFFF;
+    --brand-01: #F9F9F9;
+    --brand-02: #f7cd40;
+    --hover-timing: all 0.2s ease;
+    -webkit-font-smoothing: antialiased;
+    --pg-margin: 80px;
+    font-smoothing: antialiased;
+    scroll-behavior: smooth;
+}
+
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+svg {
+    width: 1.2em;
+    height: 1.2em;
+    stroke: var(--text-01);
+}
+
+svg.bad {
+    stroke: none;
+    color: var(--text-01);
+}
+
+svg #icon-profile.bad {
+    width: 0.5em;
+    height: 0.5em;
+}
+
+a {
+    text-decoration: none;
+    color: var(--text-01);
+}
+
+body {
+    font-family: 'Helvetica Neue', sans-serif;
+    font-weight: 500;
+    transition: var(--hover-timing);
+    background: var(--light-01);
+    width: 100vw;
+    overflow-x: hidden;
+}
+
+header {
+    height: auto;
+    position: sticky;
+    top: 0;
+    border: 1px solid var(--brand-01);
+    z-index: 100;
+}
+
+nav {
+    color: var(--text-01);
+    font-weight: 600;
+    height: 64px;
+    background: var(--light-01);
+    display: flex;
+    flex-direction: row;
+    /*align-items: center;*/
+    /*justify-content: space-between;*/
+    padding: 0 var(--pg-margin);
+    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+    z-index: 100;
+}
+
+nav ul.navigation-menu {
+    display: flex;
+    flex-direction: row;
+    flex: 1;
+    justify-content: center;
+    position: relative;
+    top: 0;
+    z-index: 1;
+    background: none;
+    width: auto;
+    position: relative;
+    align-items: center;
+    padding-left: 12.3em;
+}
+
+nav .navigation-menu a {
+    font-size: 16px;
+    text-decoration: none;
+    color: var(--text-01);
+}
+
+nav .navigation-menu > li {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+nav .navigation-menu > li > a {
+    position: relative;
+    padding: 0 20px;
+    height: 64px;
+    display: flex;
+    align-items: center;
+    justify-items: center;
+    font-weight: 500;
+}
+
+nav .navigation-menu > li > a:hover {
+    font-weight: 700;
+    background: linear-gradient(var(--light-01) 95%, var(--brand-02) 5%);
+}
+
+nav > #logo {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    grid-gap: 8px;
+    line-height: 100%;
+}
+
+nav > #utilities {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    grid-gap: 20px;
+    cursor: pointer;
+}
+
+nav > .search-container{
+    width: 490px;
+    display: block;
+    margin: 0 auto;
+}
+
+input#search-bar{
+    height: 1.2em;
+    /*border: 1px solid #D0CFCE;*/
+    outline: none;
+    border:none;
+    border-bottom: 1px solid var(--text-01);
+    padding: 5px 10px;
+    background: none;
+    /*visibility: hidden;*/
+    /*outline: none;*/
+}
+
+.search-icon{
+    position: relative;
+    float: right;
```

Comment:
What would you think about adding something along the lines of `margin-left: 10px` here to have a little bit of spacing between the search icon and the search bar?

Reply:
i dont know if its my browser but had some trouble with this. for some reason to me its not making a difference if i increase the margin. honestly i think it might be because of the way im the svg

Reply:
Before: 
![image](https://github.com/twdly/IoTBay/assets/44388767/4212eb8a-a4fd-4725-afa0-b9aa3fc03cd7)
After: 
![image](https://github.com/twdly/IoTBay/assets/44388767/7dfc6100-a248-49b6-9581-481ebf110645)
Seems to work for me with Chrome on Windows.

Reply:
is your h1 going over two lines?

Reply:
ill chuck in the margin if its working for you

Reply:
![image](https://github.com/twdly/IoTBay/assets/44388767/6e05fd7a-d6a2-45de-b61c-396e0bd6b593)
Not that I've noticed?